### ISSUE TITLE
'uploadAFile' in platforms.d.ts is not exposed for accessibility

### ISSUE THREAD

**Issue by tusharkantijoshi** [checkout/checkout-sdk-node#371]
### Environment

* Checkout SDK version: 2.3.2
* node version: v16.15.0 (npm v8.5.5)

### Description
The current codebase lacks the necessary exposure of the 'uploadAFile' functionality within the 'platforms.d.ts' declaration file. This omission is hindering accessibility and seamless integration with other components. To enhance the overall usability and maintain consistency, it's essential to expose 'uploadAFile' in 'platforms.d.ts'. This adjustment will contribute to a more cohesive and developer-friendly environment.

**Comment by armando-rodriguez-cko** [comment]
Thank you @tusharkantijoshi! we will update it asap.

**Comment by armando-rodriguez-cko** [comment]
Hi @tusharkantijoshi! We have published a new PR with the changes to updates the types. Soon we will publish a new release. Thank you.

**Comment by tusharkantijoshi** [comment]
Thank You @armando-rodriguez-cko! Looking forward to the upcoming release.

**Comment by armando-rodriguez-cko** [comment]
New release in progress @tusharkantijoshi.
